python/jsbeautifier/javascript/beautifier.py

```diff
@@ -720,7 +741,7 @@ def handle_end_block(self, current_token):
         # try inline_frame (only set if opt.braces-preserve-inline) first
         if self._flags.inline_frame and not empty_braces:
             self._output.space_before_token = True
-        elif self._options.brace_style == 'expand':
+        elif self._options.brace_style == 'expand' or self._options.brace_style == 'expand-all':
```

Comment:
New feature to have same behavior as that of brace style option with value expand.